index.js

```diff
@@ -125,44 +321,991 @@ class instance extends instance_skel {
 				cmd = 'TransportCommand&value=8';
 				break;
 			case 'load':
+			case 'loadByDrop':
 				cmd = 'GoToClip&value=' + opt.idx;
 				break;
 			case 'loop':
 				cmd = 'LoopPlay&value=' + opt.idx;
 				break;
 			case 'format':
-				let run_format = async function (cmd, timeout) {
-					return new Promise(resolve => {
-						setTimeout(function () {
-							resolve(this.run_cmd(cmd));
-						}.bind(this), timeout);
-					});
-				}.bind(this);
+				cmd = 'StorageCommand&value=4&configid=0';
```

Comment:
Why was the `run_format` `async function` removed from this?

Reply:
This was removed and moved into a preset("Stop and format"). It was more of a macro of stop and format commands. I discussed this with @onfire4g05 and we decided this was a more elegant and flexible way for users.

Reply:
k